en/peripherals/camera.md

```diff
@@ -1,18 +1,19 @@
-# Camera Trigger
+# Camera Trigger & Capture
 
-The camera trigger driver allows the use of the AUX ports to send out pulses in order to trigger a camera.
+PX4 can map outputs to trigger a camera.
 
-In addition to a pulse being sent out, the MAVLink [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) message is published containing a sequence number (i.e. the current session's image sequence number) and the corresponding timestamp.
+Cameras can also (optionally) signal PX4 at the exact moment that a photo/frame is taken using a camera capture pin.
+This allows more precise mapping of images to GPS position for geotagging, or the right IMU sample for VIO synchronization, etc.
+
+Whenever a camera is triggered, the MAVLink [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) message is published containing a sequence number (i.e. the current session's image sequence number) and the corresponding timestamp.
 This timestamp can be used for several applications, including: timestamping photos for aerial surveying and reconstruction, synchronising a multi-camera system or visual-inertial navigation.
 
-Cameras can also (optionally) use the flight controller [camera capture pin](#camera-capture) to signal the exact moment that a photo/frame is taken.
-This allows more precise mapping of images to GPS position for geotagging, or the right IMU sample for VIO synchronization, etc.
 
 ## Trigger Configuration
 
-Camera triggering is usually configured from the *QGroundControl* [Vehicle Setup > Camera](https://docs.qgroundcontrol.com/en/SetupView/Camera.html#px4-camera-setup) section.
+Camera triggering is usually configured from the *QGroundControl* [Vehicle Setup > Camera](https://docs.qgroundcontrol.com/en/SetupView/Camera.html#px4-camera-setup) section. 
 
-![Trigger pins](../../assets/camera/trigger_pins.png)
+![Trigger pins](../../assets/camera/trigger_pins.png) 
```

Comment:
FMI did you tidy up the UI here for when control allocation is used? Specifically, I would expect the hardware settings to disappear (I can't test)

![image](https://user-images.githubusercontent.com/5368500/179888530-334b8cfd-90d6-4c95-b623-66f3502fd156.png)

Reply:
Good point, no I did not look at that.

Reply:
OK, leaving that with you. If you fix create a PR or issue for this can you ping me so I can add it as a comment?